public/gameList.php

```diff
@@ -196,6 +196,18 @@ function ListGames(
 sanitize_outputs($consoleName, $requestedConsole);
 
 RenderContentStart($requestedConsole . " Games");
+
+function renderConsoleHeading(int $consoleID, string $consoleName): string
+{
+    $systemIconUrl = getSystemIconUrl($consoleID);
+
+    return <<<HTML
+        <h2 class="flex gap-x-2 items-center">
+            <img src="$systemIconUrl" alt="Console icon" width="32" height="32">
+            <span>$consoleName</span>
+        </h2>
```

Comment:
I feel like 32px header images are too similar to 32px game images:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/7523cf85-d739-45e3-b133-dec3272ba1a0)

What do you think about 24px images and h3 text?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/83b34b33-637e-43d5-9c72-85b275cc165d)

Reply:
Just realized this also affects the all-games-for-a-console list too. And that looks okay at 32px. Probably because there's a singular header. It might be the transition from list to new header that makes it seem less desirable on the dev games list.

Reply:
I agree with your assessment that 24px images and h3 text would probably look better on those dev-specific game lists.

In https://github.com/RetroAchievements/RAWeb/pull/1605/commits/381f64985e4a0bae69d6f539e804bb70c4e74572, I've modified the function so that this change is applied on those pages, but not the console-specific game list pages.